LondonBritishLibrary/orient/BLorient3716c.xml

```diff
@@ -0,0 +1,264 @@
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <desc><term key="MagicDrawing">Coloured magic picture</term> without title.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <desc>Coloured picture representing <persName ref="PRS2548Basilide"/> spearing Satan (indicated with 
+                                    <foreign xml:lang="gez">ቅዱስ፡ ፋሲለደስ፡ ዘከመ፡ ረገዞ፡ ለሰይጣን፡</foreign>) and <persName ref="PRS8065Raguel"/> 
+                                    spearing the Dragon (indicated with <foreign xml:lang="gez">ቅዱስ፡ ራጉኤል፡ ዘከመ፡ ረገዞ፡ ለከይሲ፡</foreign>). Near the 
+                                    Dragon, Satan is depicted holding the souls of the damned (indicated with <foreign xml:lang="gez">ነፍሳት፡</foreign>),
+                                    below <persName ref="PRS8065Raguel"/> and a lion indicated with <foreign xml:lang="gez">አንበሳ፡</foreign>.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d3" type="miniature">
+                                <desc>Coloured picture representing the <term key="TrueCross">Cross</term> indicated with 
+                                    <foreign xml:lang="gez">መስቀለ፡ ኢየሱስ፡</foreign>, <term key="demon">Satan</term> indicated with 
+                                    <foreign xml:lang="gez">ሰይጣን፡ ጽዋሪ፡</foreign>, the Virgin Mary indicated with 
+                                    <foreign xml:lang="gez">ጽዮን፡</foreign>, a tabot indicated with <foreign xml:lang="gez">ታቦት፡</foreign>, 
+                                    the Saviour of the World indicated with <foreign xml:lang="gez">መድሃኔ፡ ዓለም፡</foreign> under an umbrella
+                                    indicated with <foreign xml:lang="gez">ድባብ፡</foreign>. Below the Virgin Mary indicated with 
+                                    <foreign xml:lang="gez">እማህፀነ፡ አቤቶ፡ ወልደ፡ ድንግል፡</foreign> and a man armed with a gun indicated with 
+                                    <foreign xml:lang="gez">ነፍጥ፡</foreign> and a dagger indicated with <foreign xml:lang="gez">ሾቶል፡</foreign>.
+                                </desc>
+                            </decoNote>    
+                            <decoNote xml:id="d4" type="miniature">
+                                <desc><term key="MagicDrawing">Coloured magic picture</term> without title.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="PurchaseNote">Purchase note, written on the fourth picture on the bottom part of the scroll, stating
+                                        that this scroll was bought for 2 thalers.</desc>
```

Comment:
I think it is better to use `<foreign xml:lang="gez">bǝrr</foreign>` than to "translate" it with "thalers"